product_docs/docs/pgd/5/manual/index.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: Deploying manually
+navigation:
+- 01-provisioning-hosts
+- 02-install-postgres
+- 03-configuring-repositories
+- 04-installing-software
+- 05-creating-cluster
+- 06-check-cluster
+- 07-configure-proxies
+- 08-using-pgd-cli
+---
+
+At the highest level, manually deploying PGD involves the following steps:
```

Comment:
A general comment: it might be worth noting the relationship between `PGD` and `BDR`, as while the "product branding" is PGD, a lot of the underlying functionality is still `BDR`, which will be confusing for anyone not familiar with the product's evolution.

Reply:
It's confusing to people familiar with the product's evolution, too, you know. :)

This should likely be addressed at a higher level in the documentation, though.